player/modular-player/index.html

```diff
@@ -15,56 +15,19 @@ <h4 class="mt-4">Can be configured</h4>
                 <li>Dynamically during runtime</li>
             </ul>
 
-            <div id="accordion" class="accordion mt-4">
-                <div class="card">
-                    <div class="card-header collapsed" id="headingOne" data-toggle="collapse" data-target="#collapseOne"
-                         aria-expanded="false" aria-controls="collapseOne">
-                        Required Modules
-                    </div>
-                    <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion">
-                        <div class="card-body">
-                            <ul>
-                                <li>Core</li>
-                                <li>Rendering engine: MediaSource extensions (MSE) or native engine</li>
-                            </ul>
-                        </div>
-                    </div>
-                </div>
-
-                <div class="card">
-                    <div class="card-header collapsed" id="headingTwo" data-toggle="collapse" data-target="#collapseTwo"
-                         aria-expanded="false" aria-controls="collapseTwo">
-                        Optional Modules
-                    </div>
-                    <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
-                        <div class="card-body">
-                            <ul>
-                                <li>Ads</li>
-                                <li>AES encryption</li>
-                                <li>DRM</li>
-                                <li>HLS</li>
-                                <li>Smooth Streaming</li>
-                                <li>VR/360</li>
-                                <li>Chromecast</li>
-                                <li>Polyfills & Patches for legacy browsers</li>
-                                <li>DASH</li>
-                                <li>MPEG-2 TS to MP4 muxing</li>
-                                <li>WebM Support</li>
-                                <li>MP4 Support</li>
-                                <li>Side-loaded subtitle/closed caption formats (WebVTT, TTML, …)</li>
-                                <li>In-stream closed caption formats (CEA-608)</li>
-                            </ul>
-                        </div>
-                    </div>
-                </div>
-            </div>
+            <h4 class="mt-4">Get started</h4>
+            <ul>
+                <li><span style="font-size: inherit;">Get a compact introduction with our guide on <a target="_blank" href="https://developer.bitmovin.com/playback/docs/getting-started-with-the-modular-player">Getting started with the Modular Web Player</a></span></li>
```

Comment:
nit: `compact` sounds a bit strange in this context 😅 

```suggestion
                <li><span style="font-size: inherit;">Get a quick introduction with our guide on <a target="_blank" href="https://developer.bitmovin.com/playback/docs/getting-started-with-the-modular-player">Getting started with the Modular Web Player</a></span></li>
```